docs/guides/converting_and_upgrading/orbax_upgrade_guide.rst

```diff
@@ -179,6 +179,34 @@ Then, you can call ``orbax.checkpoint.AsyncCheckpointer.wait_until_finished()``
 
 For more details, read the `checkpoint guide <https://flax.readthedocs.io/en/latest/guides/training_techniques/use_checkpointing.html#asynchronized-checkpointing>`_.
 
+You can also use Orbax AsyncCheckpointer with Flax APIs through async manager. Async manager internally calls wait_until_finished(). This solution is not actively mainted and the recommedation is to use Orbax async checkpointing.
+
+For example:
+
+.. codediff::
+  :title: flax.checkpoints(async), orbax.checkpoint(async)
+  :skip_test: flax.checkpoints
+  :sync:
+
+  PURE_CKPT_DIR = '/tmp/orbax_upgrade/pure'
```

Comment:
All caps like this indicates a global variable in python, so best to avoid. Use lowercase instead.

Reply:
I see the rest of the doc uses all caps for variable names, so feel free to ignore.

Reply:
SG!